src/Parser.js

```diff
@@ -0,0 +1,56 @@
+import { REGEX, ERROR_MESSAGES } from './resources/String.js';
+
+class Parser {
+  constructor(inputString) {
+    this.inputString = inputString;
+    this.inputNums = [];
+  }
+
+  handleDefaultDelimiter() {
+    if (REGEX.DEFAULT_DELIMITER.test(this.inputString)) {
+      const parsedNums = this.inputString.match(/\d+(\.\d+)?/g);
+      this.inputNums = parsedNums.map(Number);
+      return true;
+    }
+    return false;
+  }
```

Comment:
```suggestion
  handleDefaultDelimiter() {
    if (!REGEX.DEFAULT_DELIMITER.test(this.inputString)) return false;
    const parsedNums = this.inputString.match(/\d+(\.\d+)?/g);
    this.inputNums = parsedNums.map(Number);
    return true;
  }
```

이렇게 얼리 리턴을 사용하면, if문의 코드가 줄어들어 해당 함수가 무엇을 하려는 함수인지 더 잘 나타내질 거 같습니다!

Reply:
오오 얼리리턴 방식을 적용하니 훨씬 더 잘 읽히네요.
2주차 미션 제약 조건인 depth 기준도 더 쉽게 충족할 수 있겠구요. 

감사합니다!
2주차 미션에 적용해봐야겠어요 👍👍👍👍